keras_hub/src/models/deit/deit_presets.py

```diff
@@ -2,46 +2,46 @@
 
 # Metadata for loading pretrained model weights.
 backbone_presets = {
-    "deit-base-distilled-patch16-384_imagenet": {
+    "deit_base_distilled_patch16_384_imagenet": {
         "metadata": {
             "description": (
                 "DeiT-B16 model pre-trained on the ImageNet 1k dataset with "
                 "image resolution of 384x384 "
             ),
-            "params": 86092032,
+            "params": 86_092_032,
```

Comment:
why underscore?

Reply:
for better readability, still python will interpret these as ints.